web/styles/settings.css

```diff
@@ -1601,12 +1610,12 @@ label.preferences-radio-choice-label {
 
 #edit-user-form {
     .person_picker {
-        min-width: 206px;
+        min-width: var(--modal-input-width);
```

Comment:
I guess 6px needs to be subtracted here to keep the width same as other inputs.

Reply:
And also date field is not of same size like other inputs - 
![image](https://github.com/user-attachments/assets/bf46c3fb-e18d-4b1a-8293-bb22afd05d2a)

Reply:
Oh ha the date field came from this commit (I wrote) that merged yesterday

https://github.com/zulip/zulip/commit/774e981487ee76a554cd03a436d13b8ad462b8f7#diff-fedfd23d90028ecaecf0e1a1b7319dad2efe32d6717ba99e4334c6714e80caa8R1564

Fixed to use `var(--modal-input-width)` now :)